.gitignore

```diff
@@ -7,5 +7,6 @@
 .DS_Store
 /build
 /captures
+/app/build
 
 app/src/main/java/org/familysearch/sampleapp/AppKeys.java
```

Comment:
This is a more complete .gitignore

`
# Temp files
*~
tmp/

# NetBeans
nb-configuration.xml
# nbactions.xml

# Eclipse
.settings
.classpath
.project
.mymetadata

# IDEA
*.ipr
*.iml
*.iws
.idea/
out/

# Maven
target/

#gradle
.gradle
build

# Android
classes/
bin/
gen/
gen-external-apklibs/
libs/
local.properties

# Misc.
.DS_Store
*.orig
*.hprof

# Project Specific
app/src/main/java/org/familysearch/sampleapp/AppKeys.java
`